source/vibe/inet/url.d

```diff
@@ -39,7 +39,7 @@ struct URL {
 		m_schema = schema;
 		m_host = host;
 		m_port = port;
-		m_pathString = path.toString();
+		m_pathString = urlEncode(path.toString(), "/");
```

Comment:
There isn't behaviour changing here. I just moved encoding from `localURI` to all input methods (where encoding wasn't provided).
In other words: Old implementation saved un-encoded path inside constructor and did encoding inside `localURI`, new implementation saves encoded path inside constructor and doesn't do any encoding inside `localURI`.